pyproject.toml

```diff
@@ -109,8 +112,6 @@ test-unit = "pdm run make test-unit"
 test-integration = "pdm run make test-integration"
 test-e2e = "pdm run make test-e2e"
 coverage-report = "pdm run make coverage-report"
-unit-tests-coverage-report = "pdm run make unit-test-coverage-report"
```

Comment:
I'd keep these targets, I use them from time to time

Reply:
for sure. This was not intended, good catch.